x/shared/session.go

```diff
@@ -80,3 +80,11 @@ func GetClaimWindowOpenHeight(sharedParams *sharedtypes.Params, queryHeight int6
 	claimWindowOpenOffsetBlocks := int64(sharedParams.GetClaimWindowOpenOffsetBlocks())
 	return claimWindowOpenOffsetBlocks + sessionGracePeriodEndHeight + 1
 }
+
+// GetClaimWindowCloseHeight returns the block height at which the claim window of
+// the session that includes queryHeight closes, given the passed sharedParams.
+func GetClaimWindowCloseHeight(sharedParams *sharedtypes.Params, sessionEndHeight int64) int64 {
```

Comment:
I would prefer expressing this in terms of `GetClaimWindowOpenHeight`.

Also, we should allow passing any block height `queryHeight` instead of `sessionEndHeight`